geomesa-tools/README.md

```diff
@@ -825,6 +827,27 @@ Export queries and statistics logged for raster tables by using the `querystats`
 
 #### Example command:
     geomesa queryrasterstats -u username -p password -t somerastertable -num 10
+
+### sft2avro
```

Comment:
maybe we could rename this to something like 'generateAvroSchema' so we can get away from the x2x syntax that we tried to remove from the converters.